doc/README.md

```diff
@@ -1,33 +1,31 @@
 # Building docs
 
-We currently use Sphinx for generating the API and reference
-documentation for NetworkX.
+We use Sphinx for generating the API and reference documentation.
 
-If you only want to get the documentation, note that pre-built
-versions can be found at
+Pre-built versions can be found at
 
     http://networkx.github.io/
 
 for both the stable and the latest (i.e., development) releases.
 
 ## Instructions
 
-In addition to installing NetworkX and its dependencies, install the Python
-packages need to build the documentation by entering::
+After installing NetworkX and its dependencies, install the Python
+packages needed to build the documentation by entering::
 
-    pip install -r requirements.txt
+    pip install -r requirements/doc.txt
 
-in the ``doc/`` directory.
+in the root directory.
 
 To build the HTML documentation, enter::
 
     make html
 
-in the ``doc/`` directory. If all goes well, this will generate a
-``build/html`` subdirectory containing the built documentation.
+in the ``doc/`` directory.  This will generate a ``build/html`` subdirectory
+containing the built documentation.
```

Comment:
It might be worth adding a note here about the doc build process - it's likely that the user will see some sphinx warnings from the `make html` command (especially if they didn't install the complete set of dependencies, including `pygraphviz` e.g.) but the docs will still be built successfully anyways.

Alternatively, we could fix all of the things causing warnings so a blurb here isn't necessary :)

Reply:
Let's leave it for now, but we should aim to fix all the warnings as we can before the 2.5 release.